EventListeners.js

```diff
@@ -0,0 +1,53 @@
+// Why does this bring an error?
+// import { GAME_MODE } from "./constants";
+
+
+// move basket when left or right key is pressed
+document.addEventListener('keydown', (e) => {
+    if (e.key === 'ArrowLeft') {
+        console.log('Left key was pressed')
+        // moveBasketLeft(state, 10)
+    }; 
+    
+    if (e.key === 'ArrowRight') {
+        console.log('Right key was pressed')
+        // moveBasketRight(state, 10)
+    };
+})
+
+// move basket when mouse is used
+const playArea = document.querySelector('.play-area');
+
+playArea.addEventListener('mousemove', (e) => {
+    let basketCenter = e.clientX - ( basket.width / 2 )
+    let xPos = basketCenter + 'px';
+    basketDiv.style.left = xPos;
```

Comment:
This basket's xPos state is stored as a number type. If you try to add px to a number type, like here, it results in a new value of type string. This can result in TypeErrors later if some code tries to use that state assuming it's a number when it's not (because different types behave in different ways).

So yeah, I'd remove this later when we actually hook up the mutators. And the next line basketDiv.style.left = xPos; shouldn't need to be done in the event handlers, as that's the responsibility of the rendering logic.

Reply:
Thanks Jon. Updated.  Will the 'px' get added in the rendering logic then?

Reply:
Yes it should. That's something I totally forgot the other night when writing it.